src/openrct2-cli/Cli.cpp

```diff
@@ -18,6 +18,7 @@ using namespace OpenRCT2;
  */
 int main(int argc, const char** argv)
 {
+    int32_t rc = EXIT_SUCCESS;
     int runGame = cmdline_run(argv, argc);
     core_init();
     if (runGame == 1)
```

Comment:
```suggestion
    if (runGame == EXITCODE_CONTINUE)
```

Reply:
the EXITCODE_* enums are not defined in this context, I'd have to include the CommandLine.hpp header file

Reply:
You can include that header file